clients/src/main/java/io/strimzi/common/configuration/Constants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.common.configuration;
+
+public interface Constants {
+    int DEFAULT_MESSAGES_COUNT = 10;
+    // delay cannot be 0, as the task in scheduledExecutor would overlap
+    long DEFAULT_DELAY_MS = 1;
```

Comment:
default delay in ms ... 1 ? Are you sure? Maybe you meant 1 sec, so 1000 ms?

Reply:
No, there is a comment
When you want to sent all messages without delay, you would typically set it to 0.
The thing is, `scheduleAtFixedRate` will throw exception when you set it to 0, because the tasks can overlap, which is not desired

Reply:
So actually you are sending with 1 ms delay :-) ... not with ... no delay.
I am not sure we want this option tbh.
And if you want to allow to send without delay, you should just send 1 HTTP request containing all the `records` in one array. If you send one message per HTTP request, you are actually introducing a delay anyway.

Reply:
This option is needed in our tests
But yeah, maybe sending all the messages at once, when the delay is 0 makes sense